geode-core/src/main/java/org/apache/geode/internal/ExitCode.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.apache.geode.internal;
+
+import java.util.Arrays;
+
+import org.springframework.shell.core.ExitShellRequest;
+
+
+public enum ExitCode {
+
+  // The choice of redundant values here is to be consistent with existing behavior
+  // while allowing for future extensibility. JVM_TERMINATED_EXIT(99) exists for coverage of
+  // Spring's ExitShellRequest values in fromSpring.
+  NORMAL(0),
+  FATAL(1),
+  DEPENDENCY_GRAPH_FAILURE(-1),
+  COULD_NOT_EXECUTE_COMMAND(1),
+  INVALID_COMMAND(1),
```

Comment:
I would think for each enum, the value passed to it would be distinct. What's the reason for multiple enum value have the same shellExitCode? Would this be more concise?

enum ExitCode{
  DEPENDENCY_GRAPH_FAILURE(-1),
  NORMAL(0),
  FATAL(1),
  INSTALL_FAILURE(2),
}

I am hesitant to put the exit code used only for test in here. But that's debatable.